tests/parser/test_selector_table.py

```diff
@@ -10,6 +10,437 @@
+@external
+def aQ5ELXM0F()->uint256:
+    return 86
+@external
+def aWDT9UQVV()->uint256:
+    return 87
+@external
+def a7UU40DJK()->uint256:
+    return 88
+@external
+def aH01IT5VS()->uint256:
+    return 89
+@external
+def aSKYTZ0FC()->uint256:
+    return 90
+@external
+def aNX5LYRAW()->uint256:
+    return 91
+@external
+def aUDKAOSGG()->uint256:
+    return 92
+@external
+def aZ86YGAAO()->uint256:
+    return 93
+@external
+def aIHWQGKLO()->uint256:
+    return 94
+@external
+def aKIKFLAR9()->uint256:
+    return 95
+@external
+def aCTPE0KRS()->uint256:
+    return 96
+@external
+def aAD75X00P()->uint256:
+    return 97
+@external
+def aDROUEF2F()->uint256:
+    return 98
+@external
+def a8CDIF6YN()->uint256:
+    return 99
+@external
+def aD2X7TM83()->uint256:
+    return 100
+@external
+def a3W5UUB4L()->uint256:
+    return 101
+@external
+def aG4MOBN4B()->uint256:
+    return 102
+@external
+def aPRS0MSG7()->uint256:
+    return 103
+@external
+def aKN3GHBUR()->uint256:
+    return 104
+@external
+def aGE435RHQ()->uint256:
+    return 105
+@external
+def a4E86BNFE()->uint256:
+    return 106
+@external
+def aYDG928YW()->uint256:
+    return 107
+@external
+def a2HFP5GQE()->uint256:
+    return 108
+@external
+def a5DPMVXKA()->uint256:
+    return 109
+@external
+def a3OFVC3DR()->uint256:
+    return 110
+@external
+def aK8F62DAN()->uint256:
+    return 111
+@external
+def aJS9EY3U6()->uint256:
+    return 112
+@external
+def aWW789JQH()->uint256:
+    return 113
+@external
+def a8AJJN3YR()->uint256:
+    return 114
+@external
+def a4D0MUIDU()->uint256:
+    return 115
+@external
+def a35W41JQR()->uint256:
+    return 116
+@external
+def a07DQOI1E()->uint256:
+    return 117
+@external
+def aFT43YNCT()->uint256:
+    return 118
+@external
+def a0E75I8X3()->uint256:
+    return 119
+@external
+def aT6NXIRO4()->uint256:
+    return 120
+@external
+def aXB2UBAKQ()->uint256:
+    return 121
+@external
+def aHWH55NW6()->uint256:
+    return 122
+@external
+def a7TCFE6C2()->uint256:
+    return 123
+@external
+def a8XYAM81I()->uint256:
+    return 124
+@external
+def aHQTQ4YBY()->uint256:
+    return 125
+@external
+def aGCZEHG6Y()->uint256:
+    return 126
+@external
+def a6LJTKIW0()->uint256:
+    return 127
+@external
+def aBDIXTD9S()->uint256:
+    return 128
+@external
+def aCB83G21P()->uint256:
+    return 129
+@external
+def aZC525N4K()->uint256:
+    return 130
+@external
+def a40LC94U6()->uint256:
+    return 131
+@external
+def a8X9TI93D()->uint256:
+    return 132
+@external
+def aGUG9CD8Y()->uint256:
+    return 133
+@external
+def a0LAERVAY()->uint256:
+    return 134
+@external
+def aXQ0UEX19()->uint256:
+    return 135
+@external
+def aKK9C7NE7()->uint256:
+    return 136
+@external
+def aS2APW8UE()->uint256:
+    return 137
+@external
+def a65NT07MM()->uint256:
+    return 138
+@external
+def aGRMT6ZW5()->uint256:
+    return 139
+@external
+def aILR4U1Z()->uint256:
+    return 140
+    """
+    c = get_contract(code)
```

Comment:
```suggestion
    c = get_contract(code, override_opt_level=OptimizationLevel.CODESIZE)
```

Reply:
i think we want to actually run this using the opt mode given for the test, we get coverage over all modes here: https://github.com/vyperlang/vyper/blob/0b740280c1e3c5528a20d47b29831948ddcc6d83/.github/workflows/test.yml#L81-L84